todo/src/components/TodoContainer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import TodoItem from './TodoItem';
+
+class TodoContainer extends React.Component {
+  render() {
+    return (
+      <div>
+        <h1>Todo List</h1>
+        {this.props.state.map(item => {
+          return <TodoItem text={item.value} />;
+        })}
+      </div>
+    );
+    console.log(this.props.state);
+  }
+}
+
+const mapStateToProps = state => {
+  //   console.log(state);
+  return { state: state.todoList };
```

Comment:
This looks good 👍 `mapStateToProps()` and `connect` took some time to click for me but with time Redux as a whole will start to make tons of sense.